docs/operator-config.asciidoc

```diff
@@ -0,0 +1,61 @@
+[id="{p}-operator-config"]
+== Operator configuration
+
+The operator exposes several configuration options. Unless otherwise noted, they can be set as environment variables or command line flags.
+
+
+[width="70%",valign="middle",halign="center",options="header"]
+|==========================
+|Flag |Type|Default|Description
+
+|log-verbosity |int |0 |Verbosity level of logs. -2=Error, -1=Warn, 0=Info, >0=Debug
+|enable-debug-logs |bool |false |Enables debug logs. Equivalent to `log-verbosity=1`
+|metrics-port |int |0 |Port to use for exposing metrics in the Prometheus format. Set 0 to disable
+|operator-roles |[]string |all |Roles this operator should assume. Valid values are namespace, global, webhook or all
```

Comment:
I think this needs a bit more explanation, maybe in a separate paragraph outside of this table. E.g. its important to note that `namespace` is not the same as a setting `--namespace=my-ns` and `global` will not give you a fully working operator (unless combined with another instance running with `--operator-roles=namespace`. 

Even more counterintuitively if you want to restrict the operator to one namespace you have to run it with `--operator-roles=global,namespace --namespace=my-namespace` (excluding the webhook role because I assume in single namespace mode the operator won't have permissions to register webhooks)

Finally in all likelyhood for the next release `webhook` will be effectively a NOOP (unless we fix the webhooks after #1723 )

Reply:
>Finally in all likelyhood for the next release webhook will be effectively a NOOP (unless we fix the webhooks in #1769 )

I figured I would at least document it as is and we can modify it as needed